src/node.js

```diff
@@ -388,6 +396,53 @@
       nextTickQueue.push(obj);
       tickInfo[kLength]++;
     }
+
+    // Internal callbacks that must be run when there is no more micro-tasks
+    // scheduled for this turn.
```

Comment:
I think that @petkaantonov wanted to comply with @domenic 's suggestion in the clearest way which is "run after every microtask has finished" - this demonstrates that request quite literally. I'm not sure why it'd be any more or less work except for a one function and I do think it'd be useful to be able to run an action after the microtask queue has been drained - would do you think?